app/templating/view_context.py

```diff
@@ -141,6 +142,7 @@ def build_view_context_for_calculated_summary(metadata, schema, answer_store, sc
                                                         metadata, current_location.group_instance, formatted_total),
         'title': get_question_title(rendered_block, answer_store, schema, metadata,
                                     current_location.group_instance) % dict(total=formatted_total),
+        'collapsible': _get_collapsible_final_summary(rendered_block),
```

Comment:
I don't think that the Calculated Summary should never be collapsed so should be taken out.